tests/integration_test.rs

```diff
@@ -977,3 +977,159 @@ fn test_ptr_pack_in_stack() {
         U256::from_str_radix("0x100000000000000000000000000000000", 16).unwrap()
     );
 }
+
+#[test]
+fn test_shl_asm() {
+    let bin_path = make_bin_path_asm("shl");
+    let r1 = TaggedValue::new_raw_integer(U256::from(1));
+    let r2 = TaggedValue::new_raw_integer(U256::from(2));
```

Comment:
In your shl.zasm you have

```
add 1, r0, r1
add 2, r0, r2
```
So this doesn't actually change anything (If it said for example 3 and 4, the result would not change)
The same goes for the similar tests